pubspec.yaml

```
@@ -1,15 +1,15 @@
 name: dart_random_choice
 description: A library for generating a random choice from an iterable.
-version: 0.0.2
+version: 0.0.3-nullsafety.0
```

Following https://dart.dev/null-safety/migration-guide#package-version, I'd prefer to increment the minor version

```suggestion
version: 0.1.0-nullsafety.0
```